src/partner/modules/EventList/EventListItem.tsx

```diff
@@ -59,12 +30,13 @@ export class EventListItem extends React.Component<IEventItemProps, IEventItemSt
           <ListStyled.RowData>
             <ListStyled.H1 align="left">{props.eventInfo.orderNumber}</ListStyled.H1>
             {!props.eventView && (
-              <ListStyled.MenuOptions onClick={this.showMenuOptions}>
+              <ListStyled.MenuOptions
+                onClick={e => this.setState({ showMenu: !this.state.showMenu })}
+              >
                 <ListStyled.ImgMenu src={require("../../../images/menu-icon.png")} alt="options" />
                 <ListStyled.MenuOptionsContent show={this.state.showMenu}>
                   <Link to={"events/" + props.eventInfo.id}>View Event</Link>
-                  <a onClick={this.showModalEditEvent}>Edit Event</a>
-                  <a onClick={this.showModalCancelEvent}>Cancel Event</a>
+                  <a onClick={() => this.setState({ editEvent: true })}>Edit Event</a>
```

Comment:
if you are using `class` based components always favor `methods` over `inline-functions`